arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+            "schema": {
+              "$ref": "#/definitions/ServiceDiagnosticSettingsPutParameters"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created service diagnostic setting"
+          },
+          "200": {
+            "description": "Successful request to create a service diagnostic setting"
+          },
+          "202": {
+            "description": "Accepted request to create a service diagnostic setting"
+          }
+        }
+      }
+    },
+    "/{resourceId}/skus": {
+      "get": {
+        "description": "Lists the SKU definitions for a resource",
+        "tags": [
+          "Sku"
+        ],
+        "operationId": "SkuDefinitions_List",
+        "parameters": [
+          {
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource id."
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request for a (page of the) list of Sku definitions",
+            "schema": {
+              "$ref": "#/definitions/SkuListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/{resourceId}": {
+      "patch": {
+        "description": "Updates the current SKU for a resource",
+        "tags": [
+          "Sku"
+        ],
+        "operationId": "CurrentSku_Update",
+        "parameters": [
+          {
+            "name": "resourceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The call will update the current sku of this resource id (uri)"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The api-version to send to the server"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SkuUpdateParameters"
+            },
+            "description": "The parameters to update an sku"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to update the current Sku"
```

Comment:
Since this is a PATCH we do not need more than just check for the status code.
In any case, I am going to talk to Andy in case he wants some standard schema for the the PUT. PATCH, DELETE operations.